src/components/SafeListSidebar/SafeList/SafeListItem.tsx

```diff
@@ -0,0 +1,76 @@
+import { EthHashInfo, Text, Icon } from '@gnosis.pm/safe-react-components'
+import React, { useEffect, useRef } from 'react'
+import { useHistory, generatePath } from 'react-router'
+import { sameAddress } from 'src/logic/wallets/ethAddresses'
+import { SAFE_ROUTES, LOAD_ADDRESS } from 'src/routes/routes'
+import Link from 'src/components/layout/Link'
+import ListItem from '@material-ui/core/ListItem/ListItem'
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction/ListItemSecondaryAction'
+import styled from 'styled-components'
+import { formatAmount } from 'src/logic/tokens/utils/formatAmount'
+import { useSelector } from 'react-redux'
+import { addressBookEntryName } from 'src/logic/addressBook/store/selectors'
```

Comment:
Another convention we follow is to put external dependencies on top of the file and separated with one blank line from internal imports, components...

Could you fix this and apply the same from now? Don't worry if you have more files where you already did this, we will refactor them when we find them :wink:

Reply:
I've updated all the files that have changed in this PR to follow this.